uaclient/apt.py

```diff
@@ -448,6 +452,7 @@ def run_apt_install_command(
             override_env_vars=override_env_vars,
         )
     except exceptions.APTProcessConflictError:
```

Comment:
Same suggestion here

Reply:
I think this one is covered by the logging on line `455`